data/Makefile.am

```diff
@@ -39,8 +39,34 @@ install-sample-data:
 	rm fedora-20-icons.tar.gz; \
 	cd -
 
+com.endlessm.SystemExternalApps.rules: com.endlessm.SystemExternalApps.rules.in
+	$(AM_V_GEN) $(SED)					\
+	-e "s|\@privileged_group\@|$(PRIVILEGED_GROUP)|"	\
+	-e "s|\@helper_user\@|$(EXT_APPS_HELPER_USER)|"		\
```

Comment:
Since all you're doing is substituting these 2 values, you can also just add `data/com.endlessm.SystemExternalApps.rules` to `AC_CONFIG_FILES` in `configure.ac`. Then you can drop the handling from `EXTRA_DIST` and `CLEANFILES`.